client/view/DocSection/DocSection.css

```diff
@@ -29,7 +82,7 @@
   text-transform: uppercase;
 }
 
-summary .DocSection__subHeader {
+summary .DocSection__subTitle {
```

Comment:
perhaps I am writing about this ahead of time, but it is preferable to use a selector by `.class`

 😏